tests/e2e/create_specific_url_stats_test.go

```diff
@@ -38,12 +40,12 @@ func TestUrlsRoute_CreateSpecificUrl(t *testing.T) {
 		t.Fatalf("Error unmarshalling response body: %v", err)
 	}
 	assert.Equal(t, responseBody.ID, urlResponseBody.CreatorID)
-	assert.Equal(t, "my_code", urlResponseBody.ShortURL)
+	assert.Equal(t, config.RedirPath+"my_code", urlResponseBody.ShortURL)
 
 	// ------ CHECK REDIRECT ------
 	chans := lo.Times(3, func(i int) <-chan string {
 		return lo.Async(func() string {
-			req := httptest.NewRequest("GET", "/"+urlResponseBody.ShortURL, nil)
```

Comment:
why is the slash getting dropped here from code that was already correctly working before? 🤔

Reply:
because we're giving it an absolute URL now instead of a relative one. When it was an relative URL it would automatically append it to a base URL but now we give it an absolute one so the prefix "/" is not needed anymore
Before - ShortURL was only "bsdak" so it would get transformed to "localhost:3000" + "/" + ShortURL
After ShortURL is already "localhost:3000/bsdak"
Example ShortURL for illustrative purposes

Reply:
Ah okay I got it after seeing the new response - I was just confused why it was being changed on the test side of things without that in context. 

 
> Example ShortURL for illustrative purposes  
🤣